pages/common/javadoc.md

```diff
@@ -0,0 +1,16 @@
+# javadoc
+
+> Generate Java API documentation in HTML format from source code.
+> More information: <https://docs.oracle.com/javase/9/javadoc/javadoc-command.htm>.
+
+- Generate documentation for Java source code and save the result in a directory:
+
+`javadoc -d {{path/to/directory/}} {{path/to/java_source_code}}`
```

Comment:
Is `path/to/java_source_code` a file or a directory?

Reply:
Both file and directory work.

Reply:
Ah, I see. I can't think of a better way to phrase this at the moment, so let's leave it as-is.